packages/hooks/src/useOnrampVisible.ts

```diff
@@ -0,0 +1,11 @@
+import { useStore } from '@deriv/stores';
+
+const useOnrampVisible = () => {
+    const { client } = useStore();
+    const { is_virtual, is_crypto } = client;
+    const is_onramp_visible = !is_virtual && is_crypto;
```

Comment:
The `this.filtered_onramp_providers.length > 0` logic in the old method was always `true`, Hence I've removed the dead code.